### PR TITLE
LPS-107877 BuildLang

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by adolfopa** [comment]
Just started reviewing :)

File: modules/apps/depot/depot-web/src/main/resources/META-INF/resources/item/selector/disabled.jsp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+Esto no está habilitao
```
Comment (adolfopa) [inline modules/apps/depot/depot-web/src/main/resources/META-INF/resources/item/selector/disabled.jsp]: @AliciaGarciaGarcia This kind of thing shouldn't be visible in the commit history. Can you perform a bit of commit engineering to clean up the PR?

**Pull request closed without merging**